charts/seqr/values.yaml

```diff
@@ -169,7 +171,13 @@ checkForNewSamplesJobEnabled: false
 
 redis:
   # -- enables or disables redis deployment using this chart
-  enabled: false
+  enabled: true
+
+  # -- default redis settings
+  architecture: standalone
+  auth:
+    enabled: no
```

Comment:
dang, we had `yes/no` instead of `true/false` for this and I didn't even notice!